uber/custom_tags.py

```diff
@@ -105,14 +105,6 @@ def join_and(xs):
         return ', '.join(xs)
 
 
```

Comment:
This got re-implemented in Javascript, which allowed us to dynamically switch back and forth based on whether it's a single or group registration.